AudiusProject/audius-protocol
# ClaimableTokens - Modify amount check, comments, higher bound violation test + Reward Manager missing tests

### Description


* Upper bounds check on transfer
* Reorganize helper functions
### Tests



* Higher transfer amount check
* Clarify comments and improved var names
* Create sender public (duplicate sender)
* Delete sender public duplicate sender attestation
* Delete sender public missing sender attestation
* Delete sender dupe operator attestation

### How will this change be monitored?
